includes/class-events-store.php

```diff
@@ -999,7 +618,7 @@ public static function _query_events_raw( array $args = [] ): array {
 				'validation' => fn( $status ) => self::validate_status( $status ),
 			],
 			'limit' => [
-				'default'    => 1,
+				'default'    => 100,
 				'validation' => 'is_int',
 			],
```

Comment:
The only real change to the "API" is here. Seems more fitting for a method named `_query_events_raw()` to not default to just querying 1. 100 is plenty lightweight I think, and tends to be what I went w/ in various implementations already.